dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkflowDefinitionServiceImpl.java

```
@@ -1584,6 +1588,10 @@ protected boolean checkAndImport(User loginUser,
                 putMsg(result, Status.IMPORT_WORKFLOW_DEFINE_ERROR);
                 throw new ServiceException(Status.IMPORT_WORKFLOW_DEFINE_ERROR);
             }
+            if (ReleaseState.ONLINE.equals(schedule.getReleaseState())) {
+                Project project = projectMapper.queryByCode(projectCode);
+                schedulerApi.insertOrUpdateScheduleTask(project.getId(), schedule);
+            }
```

I think we don't need this since we don't allow to copy the online schedule workflow definition. We should keep the same logic.

---

I see what you mean. We should throw an exception when the `releaseState` is online. This would prevent user confusion. 
What do you think?

---

I think we should just import workflow with offline state. And marked in the document.

---

LGTM, thanks for the suggestion!